util/collate/collate_test.go

```diff
@@ -14,23 +14,13 @@
 package collate
 
 import (
+	"reflect"
 	"testing"
 
 	. "github.com/pingcap/check"
```

Comment:
```suggestion
```

Reply:
our target is to remove pingcap/check dependency

Reply:
Done.